### PR TITLE
Deploy config

### PR DESCRIPTION
**Pull request by mafonso**
First pass and helm config definitions.

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -1,18 +1,21 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 2
+replicaCount: 1
```
Comment (matt-crowson-moj) [inline helm_deploy/values-dev.yaml:4]: Perhaps put a comment against this why setting to 1??  not a big deal if none but could potentially be an issue if changed.

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 4
+replicaCount: 1
```
Comment (matt-crowson-moj) [inline helm_deploy/values-prod.yaml:4]: Same as the dev comment for pre-prod and prod.  Very though.

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (mafonso) [reply helm_deploy/values-dev.yaml:4]: Yes, can add that. 
This are per env values, that usually will be reviewed at different stages of the project.

**Pull request merged**